README.md

```diff
@@ -6,4 +6,39 @@
 
 ### Development
 
-* Styling: Tailwind and Flowbite
\ No newline at end of file
+* Styling: Tailwind and Flowbite
+
+### Calling and Using Models
+
+Models can be called directly from the fileserver with the following url:
+
+```
+Headers: X-DreamFactory-Api-Key: {YOUR_FILESERVER_APP_API_KEY}
+GET: {FILESERVER_URL}/files/models/{MODEL_NAME}/?zip=true
+```
+An example curl request:
+
+```
+curl -X GET "http://18.183.51.71/api/v2/files/models/TintinRocket/?zip=true"  -H "X-DreamFactory-API-Key: {API_KEY_HERE}" -o myfilename.zip
+```
+
+Note that the downloaded zip will include the parent folder "models" so you will need to move over the model_name directory to your simulator assets path.
```

Comment:
I was wondering about that, is it because we may be able to download multiple models at a time in the future?

Reply:
Its a bug in Dreamfactory itself. Might take a look if i have a moment at somepoint